From a5426d3a9261939ad47427f44317c25d29e78fcc Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Mon, 8 Oct 2018 14:56:41 -0700
Subject: [PATCH 6/6] mtd: rawnand: toshiba: Remove unused mtd variables

After commit fe8b76ec9886 ("mtd: rawnand: toshiba: Pass a single
nand_chip object to the status helper"), Clang warns:

drivers/mtd/nand/raw/nand_toshiba.c:54:19: warning: unused variable
'mtd' [-Wunused-variable]
        struct mtd_info *mtd = nand_to_mtd(chip);
                         ^
drivers/mtd/nand/raw/nand_toshiba.c:68:19: warning: unused variable
'mtd' [-Wunused-variable]
        struct mtd_info *mtd = nand_to_mtd(chip);
                         ^
2 warnings generated.

The variables were moved into toshiba_nand_benand_eccstatus so they are
no longer needed in their respective functions.

I was going to submit this upstream but Miquel said he would take care
of it soon: https://lore.kernel.org/lkml/20181008095334.6850f033@xps13/

Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 drivers/mtd/nand/raw/nand_toshiba.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/mtd/nand/raw/nand_toshiba.c b/drivers/mtd/nand/raw/nand_toshiba.c
index 0549c4b59d9c..d068163b64b3 100644
--- a/drivers/mtd/nand/raw/nand_toshiba.c
+++ b/drivers/mtd/nand/raw/nand_toshiba.c
@@ -51,7 +51,6 @@ static int
 toshiba_nand_read_page_benand(struct nand_chip *chip, uint8_t *buf,
 			      int oob_required, int page)
 {
-	struct mtd_info *mtd = nand_to_mtd(chip);
 	int ret;
 
 	ret = nand_read_page_raw(chip, buf, oob_required, page);
@@ -65,7 +64,6 @@ static int
 toshiba_nand_read_subpage_benand(struct nand_chip *chip, uint32_t data_offs,
 				 uint32_t readlen, uint8_t *bufpoi, int page)
 {
-	struct mtd_info *mtd = nand_to_mtd(chip);
 	int ret;
 
 	ret = nand_read_page_op(chip, page, data_offs,
-- 
2.19.0

